src/components/Portfolio/LoanList.vue

```diff
@@ -0,0 +1,234 @@
+							</td>
+						</tr>
+						<tr v-else-if="!loans.length">
+							<td class="tw-text-center tw-text-secondary tw-px-2" colspan="7">
+								No loans found
+							</td>
+						</tr>
+						<tr
+							v-for="loan in loans"
+							:key="loan.id"
+							class="tw-border-b tw-border-tertiary"
+						>
+							<td class="tw-px-2 tw-py-2">
+								<div class="tw-flex tw-items-center">
+									<img
+										:src="loan.image.url"
+										alt="Loan image"
+										class="tw-w-[50px] tw-h-[50px] tw-mr-2"
+									>
+									<div>
+										<div class="tw-font-semibold">
+											<a
+												:href="`/lend/${loan.id}`"
+												class="tw-text-action"
+												v-kv-track-event="[
+													'portfolio', 'click', 'View borrower details', loan.name, loan.id]"
+											>
+												{{ loan.name }}
+												<div>
+													#{{ loan.id }}
+												</div>
+											</a>
+										</div>
+										<div class="tw-text-secondary">
+											{{ loan.sector?.name || '-' }}
+										</div>
+										<div class="tw-flex tw-items-center tw-text-secondary">
+											<div class="tw-w-2 tw-h-2 tw-mr-1">
+												<kv-flag
+													v-if="loan.geocode?.country?.isoCode"
+													:country="loan.geocode?.country?.isoCode"
+												/>
+											</div>
+											{{ loan.geocode?.country?.name || '-' }}
+										</div>
+										<div class="tw-text-secondary" v-if="loan.partnerName">
+											<a
+												:href="getPartnerUrl(loan.partnerId)"
+												target="_blank"
+												rel="noopener noreferrer"
+											>
+												{{ loan.partnerName }}
+											</a>
+										</div>
+									</div>
+								</div>
+							</td>
+							<td class="tw-px-2">
+								<div class="tw-text-secondary">
+									{{ getPrintableStatus(loan.status) }}
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div class="tw-mb-1">
+										${{ loan.userProperties.loanBalance.amountPurchasedByLender }}
+									</div>
+									<div class="tw-mb-1 tw-text-secondary">
+										{{ formatDate(loan.terms.disbursalDate) || '(Endpoint TBD)' }}
+									</div>
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div>
+										${{ loan.terms.loanAmount }}
+									</div>
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									{{ loan.lenderRepaymentTerm || '-' }} mos
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div class="tw-mb-1">
+										${{ loan.userProperties.loanBalance.amountRepaidToLender }} repaid to you
+									</div>
+								</div>
+							</td>
+							<td class="tw-max-w-[150px] tw-whitespace-normal tw-break-words tw-px-2">
+								<div class="tw-items-center">
+									<template v-if="loan.teams?.values?.[0]">
+										<img
+											v-if="loan.teams.values[0].image?.url"
```

Comment:
Just want to leave a comment here noting that this is still pending a backend decision on exactly where we will get the team data...  Using the [0]th element here is not the intended solution, but it does allow the component to display a dummy team for the time being.